flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -251,40 +253,40 @@ public void testPrimaryKeyFieldsAtEndOfTableSchema() {
     LocalDate dt = LocalDate.of(2022, 3, 1);
     try {
       sql(
-          "CREATE TABLE %s(id INT, data STRING NOT NULL, dt DATE NOT NULL, PRIMARY KEY(data,dt) NOT ENFORCED) "
-              + "PARTITIONED BY (data) WITH %s",
+          "CREATE TABLE %s(id INT, name STRING NOT NULL, dt DATE NOT NULL, PRIMARY KEY(name,dt) NOT ENFORCED) "
```

Comment:
Like the previous one, we can put `id` after `name` and use `id dt` as the primary key.